extension/commercial_data/README.md

```diff
@@ -0,0 +1,416 @@
+
+#### Example
+
+```json
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/datacube/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/order/v1.0.0/schema.json"
+  ],
+  "type": "Collection",
+  "id": "PLEIADES",
+  "title": "Airbus Pleiades",
+  "description": "Pleiades is a satellite constellation providing very high-resolution optical imagery and is owned by Airbus. Pléiades is composed of two twin satellites orbiting the Earth 180° apart. The satellites deliver 0.5 m optical imagery and offer a daily revisit capability to any point on the globe.",
+  "license": "proprietary",
+  "providers": [...],
+  "extent": {...},
+  "links": [...],
+  "cube:dimensions": {...},
+  "summaries": {...},
+  "assets": {...},
+  "order:status": "orderable",
+  "order_parameters": [
+      {
+        "name": "sample_type",
+        "description": "sample type of the output raster.",
+        "schema": {
+          "type": "string"
+        },
+        "enum": [
+          "UINT16",
+          "UINT8"
+        ]
+      },
+    ],
+  "pricing": {
+    "searching": null,
+    "ordering": {
+      "description": "Minimum area per order is 0.25 km2. The price is calculated based on a 6-months sliding window.",
+      "links": [
+        {
+          "title": "Airbus Pleiades pricing",
+          "rel": "related",
+          "href": "https://www.sentinel-hub.com/pricing/#TPD_pricing"
+        }
+      ]
+    }
+  }
+}
+```
+
+### Filtering parameters discovery
+
+Searching for products can support refining the search by filtering with general or collection-specific attributes. Backends should implement a top level `/queryables` endpoint for attributes available for all collections, and collection-specific attributes should be provided at `/collections/{collection-id}/queryables` according to [OGC Queryables specification](https://portal.ogc.org/files/96288#filter-queryables).
+
+#### Example
+
+Example response from `GET collections/PLEIADES/queryables`:
+```json
+{
+   "$schema":"http://json-schema.org/draft-07/schema#",
+   "$id":"http://example.com/commercial_data/PLEIADES/queryables",
+   "type":"object",
+   "title":"Queryables of Airbus Pleiades collection.",
+   "description":"Available properties for CQL filtering of products.",
+   "properties":{
+      "processing_level":{
+         "title":"Processing level",
+         "description":"When searching, you will receive results from the full catalog as well as the Living Library, which holds images that have cloud cover under 30% and Incidence angle under 40°. If you want to search only Living Library results, you will need to filter using processingLevel. This value could be equal to SENSOR (images which meet Living Library criteria) and ALBUM (images that do not meeting Living Library criteria in terms of incidence angle and cloud cover).'",
```

Comment:
To get a more concise example, maybe shorten the text to a sentence?

Reply:
Done so.